### DISCUSSION TITLE
use of EXT4 filesystems in AWS ec2 instances

### DISCUSSION THREAD

**Discussion by danie-dejager** [amazonlinux/amazon-linux-2022#230] [Q&A] [answered]
I've noticed that the ext4 filesystem is not included in any of the latter Linux releases, AMZN2 and AL2022. Was this a choice to deprecate EXT4 in AWS or is it accidently omitted? I only use EXT4 on a very old instances running Centos 7 so I would not have been affected. Should I need to mount the volume on a new  AMZN2 or AL2022 host I'll have issues though.

**Comment by stewartsmith** [comment, marked as answer]
The `ext4` file system is included in AL2 and AL2023 releases. Any volume should mount fine. It is possible that the utilities for `ext4`, namely the `e2fsprogs` package isn't on the base images though, but it can be easily installed with `dnf`
